bin/improver-generate-topographybands-ancillary

```diff
@@ -0,0 +1,98 @@
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+"""Script to run topographic bands ancillary generation."""
+
+import argparse
+import os
+import iris
+from glob import glob
+
+from improver.generate_ancillaries.generate_ancillary import (
+    GenerateOrographyBandAncils)
+from improver.generate_ancillaries.generate_ancillary import (
+     find_standard_ancil)
+
+# The following dictionary defines the orography altitude bands above/below
+#    sea level for which masks are required.
+#   'land': land points outside the altitude band are set to 0 and those
+#           within the band are set to 1.
+#   'max_land_threshold': land points above the altitude threshold are set to
+#            0 and those above the threshold are set to 1.
+#   In both cases, sea points are masked and set to the default fill_value
+THRESHOLDS_DICT = {'land': [[-500, 0], [0, 50], [50, 100], [100, 150],
+                           [150, 200], [200, 250], [250, 300], [300, 400],
+                           [400, 500], [500, 600]],
+                  'max_land_threshold': [[600]]}
+
+
+def main():
+    """Load in arguments and get going."""
+    parser = argparse.ArgumentParser(
+        description=('Read input orography and landmask fields. Mask '
+                     'data inside the bounds specified in the '
+                     'THRESHOLD_DICT dictionary' ))
+    parser.add_argument('--force', dest='force', default=False,
+                        action='store_true',
+                        help=('If True, ancillaries will be generated even '
+                              'if doing so will overwrite existing files'))
+    parser.add_argument('input_filepath_stage_orography',
+                        metavar='INPUT_FILE_STAGE_OROGRAPHY',
+                        help=('A path to an input NetCDF file (from StaGE) '
+                              'containing the orography to be processed.'))
+    parser.add_argument('input_filepath_landmask', metavar='INPUT_FILE_LAND',
+                        help='A path to an input land mask file.')
+    parser.add_argument('output_filepath', metavar='OUTPUT_FILE',
+                        help='The output path for the processed NetCDF.')
+    parser.add_argument('grid', metavar='STANDARD_GRID',
+                        help='The standard grid to process landmask on.')
```

Comment:
Removed the grid option as it has been redundant since the removal of the interpolation step.